packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -219,6 +219,40 @@ storiesOf('TableView', module)
       </TableView>
     )
   )
+  .add(
+    'horizontal scrolling only',
```

Comment:
Noticed this weird issue where the underlying background color doesn't extend the the end of the row after keyboard focusing a selected row:
<img width="375" alt="image" src="https://user-images.githubusercontent.com/9661127/166841057-2f014819-1347-440e-8e4b-e54dec8afbcb.png">
Haven't looked into it too deeply but perhaps a pre-existing issue? It seems that the last cell isn't being rendered since it is out of view but I thought we do 1.5 * the width of content to render...

Reply:
I've verified using this story in main that this is pre-existing

Reply:
It's because the padding inside the next cell is partially visible. you can't see the header either because it's got the same padding. you could see it if you turned on the dividers:
![Screen Shot 2022-05-05 at 2 21 27 PM](https://user-images.githubusercontent.com/698229/167027686-06ab8025-e782-449f-9844-6a949a56e90d.png)

Reply:
wait, now it's not doing it on main. and it's not doing it in my branch. it seems, intermittent...

Reply:
Hm weird, doesn't seem to happen as much after a reload, definitely intermittent (and doesn't really repro that easily after said reload). Not an issue for this PR IMO